### PR TITLE
Add the first version of the README.md file for Bit Blazor Components (#339)

### LINKED ISSUE #339: Add a README.md file for the Blazor Components project
In order to enhance the introduction of the Bit Blazor components, it's better to add a README.md file to the components project.

### PR DESCRIPTION
**Pull request by Cyrus-Sushiant**
This closes #339

### CODE REVIEW

**Comment by msynk** [comment]
@ysmoradi 
I'm not sure what you mean here. let's have a meeting and discuss it further.

**Comment by ysmoradi** [comment]
That's fine. Let's merge it @msynk

File: src/Client/Web/Bit.Client.Web.BlazorUI/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+[![NuGet Version](https://img.shields.io/nuget/v/Bit.Client.Web.BlazorUI.svg?style=flat)](https://www.nuget.org/packages/Bit.Client.Web.BlazorUI/) ![Nuget](https://img.shields.io/nuget/dt/Bit.Client.Web.BlazorUI.svg)
+
+BitComponent is a set of native and high performance Blazor UI controls. This is implemented in C# and this does not depend on or wrap existing JavaScript frameworks or libraries.
+
+**Why choose Bit Blazor Components?**
+- BitComponent is free
+- BitComponent is open source
+- BitComponent is high performance
+- BitComponent is native blazor components
+- BitComponent is supported in **server-side** and **client-side** (WASM) Blazor
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/README.md:11]: I think it's better to change the content just like this:
- Bit components are free
- Bit components are open source
- Bit components have a high performance
- Bit components are Blazor native components
- Bit components are supported in both **server-side** and **client-side** (WASM) Blazor

File: src/Client/Web/Bit.Client.Web.BlazorUI/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+to use BitComponent, please follow these steps:
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/README.md:13]: we need to change this line just like this:
To use the Bit components, please follow these steps:

**Review by maryamhdr — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/README.md (hunk 1 above)
Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/README.md:11]: I agree, we don't have a concept named `BitComponent` atm 😁

**Comment by msynk** [comment]
@Cyrus-Sushiant any updates?

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**